src/Handler/Command.hs

```diff
@@ -56,6 +59,9 @@ patchCommandR token = do
 getCommandR :: Token -> Handler TypedContent
 getCommandR token = do
     command <- unsafeRunStorage $ get404 token
+    mcontent <- if commandArchived command
```

Comment:
I wonder if it would be clearer to have a sum type for the data source (`ArchivedContent`/`LiveContent` or something). I had to go back and forth between this and the view where it's used to figure out what it meant to maybe have content.